etl/steps/data/grapher/artificial_intelligence/2024-06-28/ai_investment_generative_companies.meta.yml

```diff
@@ -27,8 +27,13 @@ tables:
           - In the absence of a comprehensive price index that captures the price of AI-specific goods and services, one has to rely on one of the available metrics for the price of a bundle of goods and services. In the end we decided to use the US Consumer Price Index (CPI).
           - The US CPI does not provide us with a volume measure of AI goods and services, but it does capture the opportunity costs of these investments. The inflation adjustment of this time series of AI investments therefore lets us understand the size of these investments relative to whatever else these sums of money could have purchased.
         description_key:
+          - Annual data points can be skewed by one-time events like major acquisitions, making trends hard to interpret.
+          - Year-to-year changes likely reflect broader economic conditions like interest rates and market sentiment rather than AI-specific trends.
+          - The data's methodology isn't fully clear about whether and how it captures important aspects of AI investment, including the activities of public AI-related companies (e.g., NVIDIA, TSMC), corporate internal R&D investments in AI, government funding and public sector investments, data center and infrastructure spending, hardware deployment and semiconductor manufacturing, and research and talent costs.
+          - The categories shown (mergers, private investment, public offerings, minority stakes) suggest a focus on traditional corporate finance deals, but without a detailed methodology, we can't be certain about what's included or excluded.
```

Comment:
I would put this point first, as it refers especifically to the data

Reply:
But I actually I see there is not such disaggregation of the data, so you should probably remove it